src/views/domain-page/domain-page-workflows-table-end-message/domain-page-workflows-table-end-message.tsx

```diff
@@ -34,9 +28,17 @@ export default function DomainPageWorkflowsTableEndMessage(props: Props) {
 
   if (props.hasNextPage) {
     return (
-      <div data-testid="infinite-scroll-spinner" ref={targetRef}>
+      <InView
+        as="div"
+        data-testid="infinite-scroll-spinner"
+        onChange={(inView) => {
```

Comment:
Can you test loading data that is smaller than page height.
For example, make the pages size 2 and see if it will keep loading more until the list items fill the page.
From the code it seems like it will only load one page and wait for on change to run again, which won't happen.

Reply:
I tested this case by default anyway, because 10 items is still less than the page height.
It loads correctly.

Reply:
Confirmed that this works with page size 2 as well.